.travis.yml

```diff
@@ -39,6 +39,7 @@ env:
 
 before_install:
   - &pip_install pip install --no-cache-dir "pip>=20.0.2"
+  - '[ "$TRAVIS_BRANCH" != "master" ] || export PYTEST_SENTRY_ALWAYS_REPORT=1'
```

Comment:
late to the party here, but why was this quoted?

Reply:
I copied it from the relay repo

Reply:
interesting